pkg/capabilities/kubernetes/types.go

```diff
@@ -46,3 +46,54 @@ type GetResourceRequest struct {
 	// might cause issues to the cluster
 	DisableCache bool `json:"disable_cache"`
 }
+
+// SubjectAccessReviewRequest represents an  authorization.k9s.io/v1
+// SubjectAccessReview, used by the `can_i` function.
+type SubjectAccessReviewRequest struct {
```

Comment:
Can not use the types from this package? 
https://pkg.go.dev/k8s.io/api/authorization/v1#SubjectAccessReview

Reply:
As I understand, no, because we cannot compile the package with tinygo. Tried to include the package into our github.com/kubewarden/k8s-objects, and it turned out to be more complicated than expected, just for these 2 types. Hence I decided to provide these types here.